articles/personal_growth.md

```diff
@@ -0,0 +1,33 @@
+
+Assuming that, this is the easiest way and what we will find most often within projects. People are asking for help or review, pair programming e.g. peer groups outside project to share experience and ask for advice. As this is most often related to objective questions or tasks, my experience is that most often people are happy to be linked with others to improve their working results. Most often in self-organizing teams team members switch tasks from one to each other to have best match of skills and performance for the respective task. Maybe you have also the possibility to be supported by a consulting coach or mentor.
+
+The last aspect I would like to share may be the most critical one. But it is also part of the process of growing. All of you who have a garden with trees or roses will be aware of the fact that sometimes you have to cut down old boughs and branches to enable the plant to grow any further. Even if it seems as if the plant is destroyed and will never come to life again, latest at spring it will be back again, greater and brighter than ever before (if it was not cut too much but in an accurate way). Same is valid for us human beings. Sometimes we need someone telling us where we have bad behaviors, where we follow old patterns and where we should be open for new ones.
+
+We may not be pleased to hear where we are not performing well but sometimes we have to decide if we are going to take feedback as an advice to follow or not. Any feedback should at least lead us to think about ourselves, even if we decide that what we did was ok, and we will not change anything. Possible ways to grow are for example to ask for a 360-degree feedback from team members, peers and superior. Additionally, it is key always to reflect on ourselves and on the feedback we received. Therefore, it is important that we know our own strengths and weakness and find ways to strengthen the strengths and to overcome our weaknesses. An external personal mentor or coach can also be very helpful; also, soft skills training or professional literature can help to expand your personal horizon.
+
+The conclusion of my recommendation is to check staffing of the project team right at the beginning in order to find someone to take over this mentoring role [habe ich das richtig verstanden?]. Not only the availability of the resources but also if they fit for the project. Sometimes it seems to be a good idea to have a "training on the job" setup. In this case it is important to ensure that there is someone within the project taking over the trainer role and to consider this approach inside the project planning (time and budget) to give enough space to grow otherwise it will be frustrating for all involved persons. On the other hand, it is the responsibility of each and every one of us to take the opportunity to grow by looking for our personal gardener or a growth concept as well as being prepared to be the gardener for someone else to accompany in the growth process.
```

Comment:
Die Conclusion gefällt mir.

"[habe ich das richtig verstanden?]" -> löschen

Kürzungspotential: "to give enough space to grow otherwise it will be frustrating for all involved persons" -> weglassen

"On the other hand, it is the responsibility of each and every one of us to take the opportunity to grow by looking for our personal gardener or a growth concept as well as being prepared to be the gardener for someone else to accompany in the growth process." -> super!